### ISSUE TITLE
:nth-child(3n+1) marked as "Invalid component selector" (BEM)

### ISSUE THREAD

**Issue by marienvo** [postcss/postcss-bem-linter#73]
This selector does not seem to be valid:
```css
.home__services-item:nth-child(3n+1)
```

While this one does:
```css
.home__services-item:nth-child(3n)
```

I think they should be validated the same. I'm getting this error: 
**`Invalid component selector ".home__services-item:nth-child(3n+1)" [postcss-bem-linter]`**

**Comment by davidtheclark** [comment]
Nice catch ... I believe I see the bug. Will fix sometime today.

**Comment by davidtheclark** [comment]
Fix released in 2.2.0.
